### PR TITLE
Chrome extension

### PR DESCRIPTION
**Pull request by hbazan**
Initialization for Chrome Extension contexts

### CODE REVIEW

File: libs/lz-string.js [hunk 1]
```diff
@@ -496,6 +496,8 @@ var LZString = {
 
 if (typeof define === 'function' && define.amd) {
   define(function () { return LZString; });
+} else if (typeof chrome !== "undefined" && typeof chrome.extension !== "undefined") {
```
Comment (hbazan) [inline libs/lz-string.js:499]: if we are running on chrome and on a chrome extension context, the scope is the "UnsafeWindow" where the extension is running

**Comment by pieroxy** [comment]
@Rycochet @JobLeonard @rquadling Any objection to merge this ?

**Comment by JobLeonard** [comment]
LGTM

Apologies, @hbazan, I had overlooked this in my notifications in April

**Comment by phanirithvij** [comment]
I believe this can be merged @pieroxy.

**Comment by rquadling** [comment]
I'm only 21 months late! Sorry. I'm not a JS expert, but can help deal with releasing if that's a requirement still.

**Comment by Rycochet** [comment]
Initialisation has changed and is now build via `vite` and `rollup` - I also try to use `globalThis` instead of window etc, so can you check the rc builds (not currently released to npmjs, so from here only) and see if something needs to be done for supporting extensions?

Closing this as the method will have to be very different if so!

**Pull request closed without merging**